_episodes/08-maintenance.md

```diff
@@ -68,4 +68,6 @@ Our repositories use the following labels (and colors) for issues and pull reque
     rather than the lessons themselves.
 *   `work-in-progress` (#7ae78): someone is still working on this.
 
+[swc-releases]: https://github.com/swcarpentry/swc-releases
```

Comment:
Can you move this line to `_includes/links.md`.

Reply:
Sure :)